pyvista/examples/downloads.py

```diff
@@ -4831,17 +4831,30 @@ def download_drill(load=True):  # pragma: no cover
 _dataset_drill = _SingleFileDownloadableDatasetLoader('drill.obj')
 
 
-def download_action_figure(load=True):  # pragma: no cover
+def download_action_figure(load=True, *, high_resolution=False):  # pragma: no cover
```

Comment:
Can also call this `decimate`... or some other term (coarse=True) ?

```suggestion
def download_action_figure(load=True, *, decimate=True):  # pragma: no cover
```

Note that a similar change is introduced in #7131 which also uses the term `high_resolution`, but this dataset is ImageData so `resample` is used instead of `decimate`. So a similar suggestion there may be to use the term `downsample` instead `high_resolution`.